### DISCUSSION TITLE
Pictures in the about_md are not displayed

### DISCUSSION THREAD

**Discussion by Dragon-Y233** [academicpages/academicpages.github.io#2527] [Q&A]
When I modify my code in about_md, it shows up in preview, but not on web pages
![1729044608719](https://github.com/user-attachments/assets/38d42858-04ca-49e7-b6e4-57e9837e4299)

**Comment by rjzupkoii** [comment]
That looks like the icon for a broken reference to an image. What path are you using to refer to them? Usually it should be something along the lines of `/images/image.png`

**Reply by rjzupkoii** [reply]
@logan14925 The paths don't look to be quite correct, try `/images/inverse_results.jpg`

**Reply by rjzupkoii** [reply]
@logan14925 Double check your `/images/` [directory](https://github.com/logan14925/Jiacheng_Xue.github.io/tree/master/images) to make sure the file is present as well - typically the three most common reasons for a broken image is going to be that the path is wrong, something is misspelled, or the file is not in the correct place. Beyond there are no restrictions in the template in terms of where images can appear and the [online example](https://academicpages.github.io/) has one on the homepage.
